mesheryctl/internal/cli/root/perf/perf.go

```diff
@@ -221,49 +189,43 @@ var PerfCmd = &cobra.Command{
 
 		req, err := http.NewRequest("POST", mesheryURL, bytes.NewBuffer([]byte(postData)))
 		if err != nil {
-			log.Print("Error in building the request")
-			log.Fatal("Error Message:\n", err)
-			return
+			return errors.Wrapf(err, "failed to create new request to %s", mesheryURL)
 		}
 
 		if err := AddAuthDetails(req, tokenPath); err != nil {
-			log.Printf("Error Authorizing request : %v", err.Error())
-			return
+			return errors.Wrap(err, "failed to add auth details to request")
 		}
 
 		q := req.URL.Query()
 		q.Add("name", testName)
 		q.Add("loadGenerator", loadGenerator)
-		if len(testMesh) > 0 {
+		if testMesh != "" {
 			q.Add("mesh", testMesh)
 		}
 		req.URL.RawQuery = q.Encode()
 
 		client := &http.Client{}
 		resp, err := client.Do(req)
-
 		if err != nil {
-			log.Print("\nFailed to make request to URL:", testURL)
-			log.Fatal("Error Message:\n", err)
-			return
+			log.Info("\nFailed to make request to URL:", testURL)
```

Comment:
Since we are reporting the error, I think the following log statement is not needed.

Reply:
Removed